src/menu/menuitem.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+
+@Entity('MenuItem')
+export class MenuItem {
+    @PrimaryGeneratedColumn()
+    Id: number;
+
+    @Column()
+    Name: string;
+
+    @Column()
+    Category: string;
+
+    @Column()
+    Price: number;
```

Comment:
![RUdPyQP](https://user-images.githubusercontent.com/13047857/135721855-840db73a-14c9-456c-b69f-58bd934880a5.jpg)

We use camelCase to name class properties